samples/uptime.js

```diff
@@ -243,6 +243,55 @@ function deleteUptimeCheckConfig(projectId, uptimeCheckConfigId) {
   // [END monitoring_uptime_check_delete]
 }
 
+function updateUptimeCheckConfigDisplayName(
+  projectId,
+  uptimeCheckConfigId,
+  displayName,
+  path
+) {
+  // [START monitoring_uptime_check_update]
+  // Imports the Google Cloud client library
+  const monitoring = require('@google-cloud/monitoring');
+
+  // Creates a client
+  const client = new monitoring.UptimeCheckServiceClient();
+
+  /**
+   * TODO(developer): Uncomment and edit the following lines of code.
+   */
+  // const projectId = 'YOUR_PROJECT_ID';
+  // const uptimeCheckConfigId = 'YOUR_UPTIME_CHECK_CONFIG_ID';
+  // const displayName = 'A New Display Name';
+  // const path = '/some/path';
+
+  const request = {
+    // i.e. name: 'projects/my-project-id/uptimeCheckConfigs/My-Uptime-Check
+    name: client.uptimeCheckConfigPath(projectId, uptimeCheckConfigId),
+  };
+
+  console.log(`Updating ${request.name} to ${displayName}`);
+
+  // Updates the display name and path on an uptime check config
+  request.uptimeCheckConfig = {
+    name: request.name,
+    displayName: displayName,
+    httpCheck: {path: path},
+  };
+
+  request.updateMask = {paths: ['display_name', 'http_check.path']};
+
+  client
```

Comment:
Maybe not for this PR, but should we consider migrating all of our samples to async/await now?

Reply:
Oh yes please :+1: